test/auth/authenticate.test.js

```diff
@@ -61,3 +68,55 @@ describe('getAuth', () => {
     expect(mockPublicKeyFunc).toHaveBeenCalledWith(undefined)
   })
 })
+
+describe('checkAuthGroup', () => {
+  const adminGroupId = process.env.ADMIN_AD_GROUP_ID
```

Comment:
Why get this from an env var? Doesn't this just complicate the test setup slightly, or this there a reason for not just using a value here?

Reply:
just also makes sure it is coming from env var